src/test/java/cc/robotdreams/WebTest.java

```diff
@@ -0,0 +1,55 @@
+package cc.robotdreams;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class WebTest {
+    WebDriver driver;
+
+    @BeforeTest
+    public void setup() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--start-maximized");
+
+        driver = new ChromeDriver(options);
+    }
+
+    @Test
+    public void openWebsite() throws InterruptedException {
+        driver.get("https://demoqa.com/elements");
+
+        driver.findElement(By.xpath("//li[5]/span[text()='Buttons']")).click();
```

Comment:
If your locator related to text then do not use order of `li` elements in xpath, this will be better `//ul[@class='menu-list']/li/span[text()='Buttons']`